posthog/models/filters/mixins/funnel.py

```diff
@@ -71,6 +75,42 @@ def microseconds_from_days(days):
         return microseconds * FunnelWindowDaysMixin.milliseconds_from_days(days)
 
 
+class FunnelWindowMixin(BaseParamMixin):
+    @cached_property
+    def funnel_window(self) -> Optional[int]:
+        _amt = int(self._data.get(FUNNEL_WINDOW, "0"))
+        if _amt == 0:
+            return None
+        return _amt
+
+    @cached_property
+    def funnel_window_interval(self) -> Optional[str]:
+        _interval = self._data.get(FUNNEL_WINDOW_INTERVAL, None)
+        return _interval
+
+    @include_dict
+    def funnel_window_to_dict(self):
+        return {FUNNEL_WINDOW: self.funnel_window} if self.funnel_window else {}
+
+    @include_dict
+    def funnel_window_interval_to_dict(self):
+        return {FUNNEL_WINDOW_INTERVAL: self.funnel_window_interval} if self.funnel_window_interval else {}
+
+    def funnel_window_interval_ch(self) -> str:
+        if self.funnel_window_interval == "minute":
```

Comment:
I think we can be a bit more flexible here, and allow `Minute`, `MINUTE`, `mInuTe` ? So that:
```suggestion
        if self.funnel_window_interval.lower() == "minute":
```